src/sourmash/search.py

```diff
@@ -11,6 +11,25 @@
 from .sketchcomparison import FracMinHashComparison, NumMinHashComparison
 
 
+def calc_threshold_from_bp(threshold_bp, scaled, query_size):
+    """
+    Convert threshold_bp (threshold in estimated bp) to
+    fraction of query & minimum number of hashes needed.
+    """
+    threshold = 0.0
+    n_threshold_hashes = 0
+
+    if threshold_bp:
+        # if we have a threshold_bp of N, then that amounts to N/scaled
+        # hashes:
+        n_threshold_hashes = float(threshold_bp) / scaled
```

Comment:
good catch :)

it's not constrained in this PR, which is mostly about adding tests so that I can refactor the code - but as [you noted](https://github.com/sourmash-bio/sourmash/pull/2116/files#r922571983) it's handled over in #2116!